api/testutils/kusttest/harnessenhanced.go

```diff
@@ -41,6 +41,7 @@ func MakeEnhancedHarness(t *testing.T) *HarnessEnhanced {
 	pte := newPluginTestEnv(t).set()
 
 	pc, err := konfig.EnabledPluginConfig(types.BploLoadFromFileSys)
+	pc.FnpLoadingOptions.EnableStar = true
```

Comment:
Is this ok? Is there a better way to get this set, perhaps for only my new test?

Reply:
It's gratifying to start getting starlark tests / examples in place!

This  is fine.  We can always rejigger as we get more starlark.  The plugin config maker could adopt a chaining mutator pattern - `konfig.MakePluingConfig().EnableThis().EnableThat().`.  Or just do the Go idoidom of declaring a struct and setting fields inline.